niworkflows/viz/utils.py

```diff
@@ -199,20 +199,45 @@ def extract_svg(display_object, dpi=300, compress='auto'):
 def cuts_from_bbox(mask_nii, cuts=3):
     """Finds equi-spaced cuts for presenting images"""
     from nibabel.affines import apply_affine
-    mask_data = mask_nii.get_data()
-    B = np.argwhere(mask_data > 0)
-    start_coords = B.min(0)
-    stop_coords = B.max(0) + 1
+
+    mask_data = mask_nii.get_data() > 0.0
+
+    # First, project the number of masked voxels on each axes
+    ijk_counts = [
+        mask_data.sum(2).sum(0),  # project to i (sagittal)
```

Comment:
I see, this is sagittal in the sense that the longitudinal-vertical plane is sagittal and so you're collapsing each sagittal plane onto the transverse (i) axis. If that's the case, and we're working with transverse-longitudinal-vertical images, then, we should be collapsing the `j` and `k` axes:

```suggestion
        mask_data.sum(2).sum(1),  # project sagittal planes to transverse (i) axis
```